contracts/action-handlers/AddPrepaidCardSKUHandler.sol

```diff
@@ -0,0 +1,95 @@
+pragma solidity ^0.8.9;
+pragma abicoder v1;
+
+import "../core/Ownable.sol";
+import "../core/Versionable.sol";
+import "../token/IERC677.sol";
+import "../PrepaidCardManager.sol";
+import "../PrepaidCardMarketV2.sol";
+import "../TokenManager.sol";
+import "../IPrepaidCardMarket.sol";
+import "../VersionManager.sol";
+
+contract AddPrepaidCardSKUHandler is Ownable, Versionable {
+  address public actionDispatcher;
+  address public prepaidCardManagerAddress;
+  address public tokenManagerAddress;
+  address public versionManager;
+
+  event Setup();
+
+  function setup(
+    address _actionDispatcher,
+    address _prepaidCardManager,
+    address _tokenManagerAddress,
+    address _versionManager
+  ) external onlyOwner returns (bool) {
```

Comment:
Need to add a configuration for this contract in https://github.com/cardstack/card-pay-protocol/tree/main/scripts/deploy/config/default

Reply:
Additionally you'll need to configure ActionDispatcher for this new action too https://github.com/cardstack/card-pay-protocol/blob/main/scripts/deploy/config/default/ActionDispatcher.ts